src/index.ts

```diff
@@ -250,23 +250,8 @@ function startApp(appSchema, path: string) {
 
 const app = express()
 
-let appV1: express.Express
-let appV2: express.Express
-
-app.all("/", (req, res, next) => {
-  if (!appV1) {
-    appV1 = express()
-    appV1.use("/", startApp(schemaV1, "/"))
-  }
-  appV1(req, res, next)
-})
-
-app.all("/v2", (req, res, next) => {
-  if (!appV2) {
-    appV2 = express()
-    appV2.use("/", startApp(schemaV2, "/v2"))
-  }
-  appV2(req, res, next)
-})
+// This order is important for dd-trace to be able to find the nested routes.
+app.use("/v2", startApp(schemaV2, "/"))
```

Comment:
This feels nicer than the more roundabout way it was being declared (not sure exactly why that was?).

Although, `startApp` feels a _bit_ misnamed now (outside the scope of this PR I think).

Reply:
Ha, primarily my lack of express knowledge it seems. 

Agree though, looks so much better now 👍.